doc/source/cd/background.ipynb

```diff
@@ -0,0 +1,608 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The test data has drifted into a previously unseen region of feature space, and the model is now misclassifying a number of test observations. If true test labels are avaiable, this is easily detectable by monitoring the test accuracy. However, labels aren't always avaiable during deployment, in which case a drift detector monitoring the covariates comes in handy. In this case, the trained MMD detector successfully detects the covariate drift.\n",
+    "\n",
+    "In a similar manner, prior drift can be monitored by training a detector on training labels, and then feeding it a model's predicted labels:\n",
+    "\n",
+    "```python\n",
+    "label_detector = MMDDrift(y_train.reshape(-1,1), backend='tensorflow', p_val=.05)\n",
+    "y_pred = clf.predict(X_test)\n",
+    "label_detector.predict(y_pred.reshape(-1,1))\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "source": [
+    "#### Concept drift \n",
+    "\n",
+    "Concept drift detectors are coming soon!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Learned drift detection <a class=\"anchor\" id=\"learned\"></a>\n",
+    "\n",
+    "\n",
+    "\n",
+    "- [Classifier](methods/classifierdrift.ipynb)\n",
+    "- [Spot the difference](methods/spotthediffdrift.ipynb)\n",
+    "- [Learned Kernel](methods/learnedkerneldrift.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Online drift detection <a class=\"anchor\" id=\"online\"></a>\n",
+    "\n",
+    "So far, we have discussed drift detection in an *offline* context, with the entire dataset $\\{\\mathbf{X}_i\\}_{i=1}^{N}$ compared to the reference dataset $\\{\\mathbf{X}^{ref}_i\\}_{i=1}^{M}$. However, during model deployment data usually arrives sequentially, and drift detection must be performed in an *online* fashion. \n",
+    "\n",
+    "<div style=\"text-align: center\">\n",
+    "<img src=\"images/online.gif\" width=\"350\"/>\n",
+    "</div>\n",
+    "\n",
+    "One approach is to perform a test for drift every $W$ time-steps, using the $W$ samples that have arrived since the last test. In other words, that is to compare $\\{\\mathbf{X}_i\\}_{i=t}^{t+W}$ to $\\{\\mathbf{X}^{ref}_i\\}_{i=1}^{M}$. Such a strategy could be implemented using any of the offline detectors implemented in alibi-detect, but being both sensitive to slight drift and responsive to severe drift is difficult. If the window size $W$ is too large the delay between test-points hampers responsiveness to severe drift, but an overly small window is unreliable. This is demonstrated below, where the offline [MMD detector](methods/mmddrift.ipynb) is used to monitor drift in data sampled from a normal distribution $\\mathcal{N}\\left(\\mu,\\sigma^2 \\right)$, with the mean starting to drift from $\\mu=0$ to $\\mu=0.5$ at $t=40$.\n",
```

Comment:
"delay between test-points" -> something like "the increased time passed between consecutive statistical tests" since test-points could be confused with points of the test set